turbinia/workers/psort.py

```diff
@@ -38,15 +72,20 @@ def run(self, evidence, result):
     """
     config.LoadConfig()
 
-    psort_file = os.path.join(self.output_dir, '{0:s}.csv'.format(self.id))
+    psort_file = os.path.join(self.output_dir, '{0:s}'.format(self.id))
```

Comment:
Why don't we want the .csv extension here?

Reply:
I think the idea was to allow for more output formats with the extension in this case not contributing to further processing.

Reply:
I'm not sure I understand, but given that the output evidence type for this is a `PlasoCsvFile`, I think it would be good to keep the .csv extension for now.   If we want to add other output formats we'll need to change the Evidence type as well, and we can probably do that in a separate PR.